File: 7KYU/Simple Fun #3: Late Ride/solution.clj
(ns kata.late-ride)
(defn late-ride [n]
  (let [hours (quot (- n (rem n 60)) 60)
        minutes (rem n 60)]
    (+
     (reduce + (->> hours str (map (comp read-string str))))
     (reduce + (->> minutes str (map (comp read-string str)))))))

File: 7KYU/Simple Fun #3: Late Ride/solution.go
package kata

func SumDigits(n int) int {
	res := 0
	for {
		if n == 0 {
			break
		}

		res = res + (n % 10)
		n /= 10
	}
	return res
}

func LateRide(n int) int {
	hours := n / 60
	n = n - (hours * 60)

	return SumDigits(n) + SumDigits(hours)
}
